### ISSUE TITLE
singular-4.3.2_p15_noexpect.patch fails to completely patch singular.pxd

### ISSUE THREAD

**Issue by strogdon** [cschwan/sage-on-gentoo#781]
```
>>> Source unpacked in /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sagemath-standard-9999/work
>>> Preparing source in /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sagemath-standard-9999/work/sagemath-standard-9999 ...
 ^[[32m*^[[0m Applying singular-4.3.2_p15_noexpect.patch ...
patching file sage/interfaces/singular.py
patching file sage/libs/singular/decl.pxd
patching file sage/libs/singular/function.pyx
patching file sage/libs/singular/ring.pyx
patching file sage/libs/singular/singular.pxd
Hunk #2 FAILED at 29.
1 out of 2 hunks FAILED -- saving rejects to file sage/libs/singular/singular.pxd.rej
patching file sage/libs/singular/singular.pyx
patching file sage/rings/polynomial/multi_polynomial_libsingular.pyx
patching file sage/rings/polynomial/term_order.py
```
singular.pxd.rej:
```
--- sage/libs/singular/singular.pxd
+++ sage/libs/singular/singular.pxd
@@ -29,6 +29,7 @@ cdef object   si2sa_ZZmod(number *n, ring *_ring, object base)
 cdef object   si2sa_NF(number *n, ring *_ring, object base)

 cdef object si2sa_intvec(intvec *v)
+cdef object si2sa_bigintvec(bigintmat *v)

 # dispatches to all the above.
 cdef object si2sa(number *n, ring *_ring, object base)
```
It looks like there is a `noexcept` at the end of the line to be patched in `singular.pyx`, i.e.
`cdef object si2sa_intvec(intvec *v) noexcept`.

**Comment by kiwifb** [comment]
I did a rookie mistake when I tested the patch from the vbaun branch to the master branch. I made it harder to so those kinds of tests when I made it easier to merge back and forth. I'll look into it shortly.

**Comment by kiwifb** [comment]
Should be fixed by 7539eea1611c97aee6efaa61d4d185257d89c88c along with other stuff that was removed too early.

**Comment by strogdon** [comment]
Fixed, thanks.
